conf/core/ConfigOptions.asciidoc

```diff
@@ -1208,6 +1248,74 @@ See also:
 
 The way subline matcher to use when determining matching sublines with generic line conflation.
 
+=== generic.point.match.threshold
+
+* Data Type: double
+* Default Value: `${conflate.match.threshold.default}`
+
+The threshold at which a match is called a match for generic points.
+
+See also:
+
+ * _Estimate Pairwise Relationships_, <<hootalgo>>
+
+=== generic.point.miss.threshold
+
+* Data Type: double
+* Default Value: `${conflate.miss.threshold.default}`
+
+The threshold at which a miss is called a miss for generic points.
+
+See also:
+
+ * _Estimate Pairwise Relationships_, <<hootalgo>>
+
+=== generic.point.review.threshold
```

Comment:
Note that some of these options like this never get used, but I was just adding them for consistencies sake. Opened #3721 to clean this type of thing up.